app/assets/javascripts/dashboard/protocol_form.js.coffee

```diff
@@ -173,7 +173,7 @@ $(document).ready ->
 
   ###END PUBLISH IN EPIC BUTTON STATES###
 
-  if $("input[name='protocol[selected_for_epic]']").val() == 'true'
+  if $("input[name='protocol[selected_for_epic]'][value='true']").parent().hasClass('active')
```

Comment:
Same, I'd much rather see CSS selectors used for this, instead another javascript selector to go to the parent and look at it's class. I think Jay's suggestion would also technically be a tiny bit faster to execute.